### PR TITLE
Added lesson_07

### PR DESCRIPTION
**Pull request by MarianaKhl**

### CODE REVIEW

File: lesson_07/homework_07.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+Код майже готовий, треба знайти помилки та випраавити\доповнити.
+"""
+def multiplication_table(number):
+    # Initialize the appropriate variable
+    multiplier = 1
+
+    # Complete the while loop condition.
+    while True:
+        result = number * multiplier
+        if  result <= 25:
```
Comment (edanilovets) [inline lesson_07/homework_07.py]:
```suggestion
        if result <= 25:
```

File: lesson_07/homework_07.py [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+            print(f"{number} x {multiplier} = {result}")
+            multiplier += 1
+        else:
+            break
+
+multiplication_table(3)
+
+print("#"*100)
```
Comment (edanilovets) [inline lesson_07/homework_07.py]:
```suggestion
print("#" * 100)
```

File: lesson_07/homework_07.py [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+# task 2
+"""  Написати функцію, яка обчислює суму двох чисел.
+"""
+def result_sum(number1, number2):
+    sum = number1 + number2
+    print(sum)
+result_sum(10,12)
```
Comment (edanilovets) [inline lesson_07/homework_07.py]:
```suggestion
result_sum(10, 12)
```

File: lesson_07/homework_07.py [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+# task 5
+"""  Написати функцію, яка приймає список слів та повертає найдовше слово у списку.
+"""
+
+def long_word(sentance):
+    if len(sentance) == 0:
+        return f"The sentence are absent"
+    words = sentance.split()   # is the list of words obtained after splitting the sentence with split()
+    longest_word = max(words, key= len)   # calculates the length of each word
```
Comment (edanilovets) [inline lesson_07/homework_07.py]:
```suggestion
    longest_word = max(words, key=len)   # calculates the length of each word
```

File: lesson_07/homework_07.py [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+    if len(sentence) == 0:
+        return f"The sentence are absent"
+
+    reverce_line = sentence[::-1]
+    print(f"Vice versa result: {reverce_line}")
+
+sentance = "Hi! How are you?"
+vice_versa_result(sentance)
+
+print("#"*100)
```
Comment (edanilovets) [inline lesson_07/homework_07.py]:
```suggestion
print("#" * 100)
```

File: lesson_07/homework_07.py [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+"""
+
+def result_average(numbers):
+    if len(numbers) == 0:
+        return f"The list is empty"
+
+    average_result = sum(numbers) / len(numbers)
+    print(average_result)
+
+numbers = [1,6,33]
```
Comment (edanilovets) [inline lesson_07/homework_07.py]:
```suggestion
numbers = [1, 6, 33]
```

File: lesson_07/homework_07.py [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+
+print("#"*100)
+
+# task 4
+"""  Написати функцію, яка приймає рядок та повертає його у зворотному порядку.
+"""
+
+def vice_versa_result(sentence):
+    if len(sentence) == 0:
+        return f"The sentence are absent"
```
Comment (edanilovets) [inline lesson_07/homework_07.py]:
```suggestion
        return f"The sentence is empty"
```

File: lesson_07/homework_07.py [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+
+    reverce_line = sentence[::-1]
+    print(f"Vice versa result: {reverce_line}")
```
Comment (edanilovets) [inline lesson_07/homework_07.py]:
```suggestion
    print(f"Reversed result: {reverce_line}")
```

File: lesson_07/homework_07.py [hunk 9]
```diff
@@ -0,0 +1,185 @@
...
+# task 9
+
+""""  # Напишіть цикл, який буде вимагати від користувача ввести слово, в якому є літера "h"
+# (враховуються як великі так і маленькі). Цикл не повинен завершитися,
+# якщо користувач ввів слово без букви "h".
+"""
+def search_h_letter():
+
+    while True:
+        input_string = input("Enter the any word with letter 'h' , please: ").lower()
```
Comment (edanilovets) [inline lesson_07/homework_07.py]:
```suggestion
        input_string = input("Enter any word with letter 'h' , please: ").lower()
```

**Comment by MarianaKhl** [comment]
Hi!

Added changes to the lesson_07/homework-07:
https://github.com/MarianaKhl/Lesson_2-GitHub-/pull/10

**Review by edanilovets — APPROVED** [review]

**Pull request merged**